### ISSUE TITLE
Hello, I have some questions about the data set of the fake news detection article

### ISSUE THREAD

**Issue by qiljj** [entitize/Fakeddit#21]
First of all, for the microblog data set, I copied the picture name in the microblog picture and searched the training set, verification set and test set in the data, but there was no corresponding picture name. I wonder, do the pictures in this website correspond to the pictures in your trained data set label? Weibo:  [https://github.com/yaqingwang/EANN-KDD18](https://github.com/yaqingwang/EANN-KDD18)

Secondly, according to this website, Fakeddit: [https://github.com/entitize/Fakeddit] (https://github.com/entitize/Fakeddit), only "image_downloader. Py" in the making. How can I get this data set without images?

I really hope you can reply me. Thank you very much.
